app/src/main/java/com/epfl/beatlink/ui/map/GoogleMapView.kt

```diff
@@ -70,14 +70,16 @@ fun GoogleMapView(
               Marker(
                   state = MarkerState(position = location),
                   title = "You are here",
-                  icon = getBitmapDescriptorFromDrawableResource(R.drawable.me_position, context),
+                  icon =
+                      getBitmapDescriptorFromDrawableResource(
+                          R.drawable.me_position, context, 100, 100),
                   anchor = Offset(0.5f, 0.5f))
               Circle(
                   center = location,
                   radius = radius,
-                  strokeColor = PrimaryPurple,
+                  strokeColor = CircleStrokeColor,
```

Comment:
strokeColor should be defined with MaterialTheme so it changes color when in dark mode:
I suggest you add in Theme.kt and use the "onBackgroud" that is not used yet.